packages/preview/light-report-uia/0.1.0/template/main.typ

```diff
@@ -0,0 +1,61 @@
+#import "@preview/light-report-uia:0.1.0": *
+
+// CHANGE THESE
+#show: report.with(
+  title: "New project",
+  authors: (
+    "Lars Larsen",
+    "Lise Lisesen",
+    "Knut Knutsen",
+  ),
+  group_name: "Group 14",
```

Comment:
The idiomatic casing convention in Typst is `kebab-case`, so `group-name`. I would recommend switching to that, but it is not a strict requirement.

Reply:
Will fix next version